addons/pos_gift_card/static/src/js/GiftCardPopup.js

```diff
@@ -4,6 +4,8 @@ odoo.define("pos_gift_card.GiftCardPopup", function (require) {
   const { useState, useRef } = owl.hooks;
   const AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
   const Registries = require("point_of_sale.Registries");
+  const core = require('web.core');
+  const _t = core._t;
```

Comment:
No need to import this. You have access to `_t` in every owl.Component via `this.env`.